cmd/internal/apidump/apidump.go

```diff
@@ -106,22 +107,18 @@ var Cmd = &cobra.Command{
 		}
 
 		// Allow specification of an alternate rotation time, default 1h.
+		// We can rotate the trace if we're sending the output to a cloud-based trace.
 		// But, if the trace name is explicitly given, or selected by tag,
 		// or we're sending the output to a local file, then we cannot rotate.
 		traceRotateInterval := time.Duration(0)
-		if outFlag.AkitaURI != nil {
+		if (outFlag.AkitaURI != nil && outFlag.AkitaURI.ObjectName == "") || projectID != "" || postmanCollectionID != "" {
```

Comment:
Combine separate checks for `outFlag.AkitaURI` and `outFlag.AkitaURI.ObjectName`, since we won't be using this flag now. So no need to throw a separate error.
The main aim is to handle `collectionId` and `projectId` for trace rotation